ActionScript/ActionScript.sublime-syntax

```diff
@@ -12,87 +12,888 @@ first_line_match: |-
+            | a(?:n(?:e(?:M(?:inimums|aximums)|Height|Title|Width))?|rentNode)
+            | r(?:operty(?:Name|Data)?|efer(?:ences|red(?:Height|Width)))
+            )
+          | E(?:n(?:dIndex|abled)|ditingData|x(?:panderSymbolName|andNodeTrigger))
+          | V(?:iewed(?:Pods|Applications)|olume|ersion|alue(?:Source)?)
+          | F(?:
+              i(?:eld|rst(?:DayOfWeek|VisibleNode))
+            | o(?:ntList|cus)
+            | ullYear
+            | ade(?:InLength|OutLength)
+            | rame(?:Color|Width)
+            )
+          | Width
+          | L(?:ine(?:Color|Weight)|o(?:cal|adTarget)|ength|a(?:stTabIndex|bel(?:Source)?))
+          | A(?:
+              s(?:cii|Boolean|String|Number)
+            | n(?:yTypedValue|imation)
+            | ctiv(?:eState(?:Handler)?|ateHandler)
+            | utoH(?:ideScrollBar|eight)
+            | llItems
+            | gent
+            )
+          )?
+        )?
+      | lobal(?:StyleFormat|ToLocal)?
+      | ain
+      | roupName
+      )
+    | x(?:updatePackety|mlDecl)?
+    | m(?:
+        y(?:MethodName|Call)
+      | in(?:imum)?
+      | o(?:
+          nthNames
+        | tion(?:TimeOut|Level)
+        | de(?:lChanged)?
+        | use(?:Move|O(?:ut|ver)|Down(?:Somewhere|Outside)?|Up(?:Somewhere)?|WheelEnabled)
+        | ve(?:To)?
+        )
+      | u(?:ted|lti(?:pleS(?:imultaneousAllowed|elections)|line))
+      | e(?:ssage|nu(?:Show|Hide)?|th(?:od)?|diaType)
+      | a(?:nufacturer|tch|x(?:scroll|hscroll|imum|HPosition|Chars|VPosition)?)
+      | b(?:substring|chr|ord|length)
+      )
+    | b(?:
+        ytes(?:Total|Loaded)
+      | indFormat(?:Strings|Function)
+      | o(?:ttom(?:Scroll)?|ld|rder(?:Color)?)
+      | u(?:tton(?:Height|Width)|iltInItems|ffer(?:Time|Length)|llet)
+      | e(?:foreApplyUpdates|gin(?:GradientFill|Fill))
+      | lockIndent
+      | a(?:ndwidth|ckground(?:Style|Color|Disabled)?)
+      | roadcastMessage
+      )
+    | onHTTPStatus
+    )
+  constants: |-
+    (?x:
+      __proto__ | __resolve
+    | _accProps | _alpha
+    | _changed | _currentframe
+    | _droptarget
+    | _flash | _focusrect | _framesloaded
+    | _global
+    | _height | _highquality
+    | _level | _listeners | _lockroot
+    | _name
+    | _parent
+    | _quality
+    | _root | _rotation
+    | _soundbuftime
+    | _target | _totalframes
+    | _url
+    | _visible
+    | _width
+    | _x | _xmouse | _xscale
+    | _y | _ymouse | _yscale
+    )
+  keywords: |-
+    (?x:
+      break
+    | case | continue
+    | do | dynamic
+    | else | extends
+    | for
+    | if | implements | import | in | interface
+    | new
+    | private | public
+    | return
+    | static | super | switch
+    | var
+    | while
+    )
+  types: |-
+    (?x:
+      Boolean
+    | Number
+    | String
+    | Void
+    )
+  constant_numeric_exponent: |-
+    (?x:
+      [eE]
+      [-+]?
+      \d+
+    )
+  constant_numeric_suffix: |-
+    (?x:
+      L|l|UL|ul|u|U|F|f
+    )
```

Comment:
Maybe also add space around pipes and sort by capital/non-captial letters (U, u)

```suggestion
    (?x: L | l | UL | ul | U | u | F | f )
```